runner/kubectl.go

```diff
@@ -109,6 +109,36 @@ func (k *Kubectl) DeleteAll(resource string, extraArgs ...string) error {
 	return k.runner.Run(k.command(args...))
 }
 
+// DeleteAllTimeout runs `sops exec-file KUBECONFIG 'kubectl delete RESOURCE -A --all --timeout=TIMEOUTs EXTRAARGS...'
+func (k *Kubectl) DeleteAllTimeout(resource string, timeout int, extraArgs ...string) error {
```

Comment:
Could you add some tests for this? Looks like there is _at least_ three cases here, OK, exit 1 timeout and exit 1 no timeout?